corehq/apps/locations/templates/locations/location_types.html

```diff
@@ -7,6 +7,8 @@
 {% block page_content %}
   {% initial_page_data 'location_types' location_types %}
   {% initial_page_data 'commtrack_enabled' commtrack_enabled %}
+  {% initial_page_data 'suggest_orphan_case_alerts_setting' suggest_orphan_case_alerts_setting %}
+  {% initial_page_data 'project_settings_url' project_settings_url %}
```

Comment:
I had to pass the `project_settings_url` in, since I cannot use `{% url ...%}` in a `blocktrans` block.